articles/vmware/vmw-how-access-vcloud-api.md

```diff
@@ -108,26 +108,26 @@ To start interacting with the vCloud API, you first need to obtain an `x-vcloud-
 
     ![Headers section with Accept header](images/vmw-restclient-accept-header.png)
 
-12. The RESTClient has now got all the required settings in place to make a request to the vCloud API to obtain a `x-vcloud-authorization` token, so click the **Send** button.
+12. The RESTClient has now got all the required settings in place to make a request to the vCloud API to obtain a `x-vmware-vcloud-access-token` token, so click the **Send** button.
 
-13. When a response is received the **Headers** tab in the *Response* section will be populated, including an `x-vcloud-authorization` token.
+13. When a response is received the **Headers** tab in the *Response* section will be populated, including an `x-vmware-vcloud-access-token` token.
 
     ![API response with authentication token](images/vmw-restclient-authentication-token.png)
```

Comment:
image looks a little odd

Reply:
I removed some of the image because it's an access token that shouldn't be public

Reply:
Sorry, could you just take another look at the image - I can see bits of the access token, and they seem to be laid over the attribute names so it looks weird

Reply:
I've uploaded a new version that uses blur.

Reply:
That looks better, thanks!